src/components/TextEditor/components/TextEditor.js

```diff
@@ -98,6 +112,87 @@ class TextEditor extends React.Component {
     this.setState({ currentLine: line });
   };
 
+  renderImageModal = () => {
```

Comment:
I assume this should be `renderForkModal`?

Reply:
Still haven't gotten around to reviewing this, but from what I saw when I was looking over it earlier this week, I think it's just a typo since the fork button is based off of our deleteSketchModal.